frontend/src/metabase/querying/notebook/components/SummarizeStep/SummarizeStepHeader/SummarizeStepHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { t } from "ttag";
+
+import { Box, Flex } from "metabase/ui";
+
+import type { NotebookStepHeaderProps } from "../../../types";
+import { NotebookStepHeader } from "../../NotebookStep/NotebookStepHeader";
+
+export function SummarizeStepHeader({
+  step,
+  title,
+  color,
+  canRevert,
+  onRevert,
+}: NotebookStepHeaderProps) {
+  const isMetric = step.question.type() === "metric";
+  if (!isMetric) {
+    return (
+      <NotebookStepHeader
+        step={step}
+        title={title}
+        color={color}
+        canRevert={canRevert}
+        onRevert={onRevert}
+      />
+    );
+  }
+
+  return (
+    <Flex c={color} fw="bold" mb="sm" gap="md">
```

Comment:
The header uses 1rem gap and the blocks below use 0.5rem gap.

![image](https://github.com/user-attachments/assets/64c404f2-8ac4-4ebc-96b9-b4b4408c683c)

Reply:
Fixed